lib/active_merchant/billing/gateways/pay_trace.rb

```diff
@@ -179,7 +176,7 @@ def build_purchase_request(money, payment_or_customer_id, options)
       def build_capture_request(money, authorization, options)
         post = {}
         post[:transaction_id] = authorization
-        add_amount(post, money, options) if options[:include_capture_amount] == true
+        add_amount(post, money, options) if money.class == Integer
```

Comment:
Why do we want to specifically check the .class here?

Reply:
When I was writing tests, if an empty string was passed through for `money` then it would give an error saying it needed to be a positive integer so I added this in to bypass the `add_amount` method in those cases. Based on how `core` handles capture and refund requests, AM will never receive an empty string so I have removed it and updated tests! 🎉